.travis.yml

```diff
@@ -31,40 +31,50 @@ matrix:
           os: linux
           dist: xenial
           env: TOXENV=py38
-        - python: 3.5
+        - python: "3.5"
           os: linux
           dist: xenial
           env: TOXENV=flake8
         - language: generic
           os: osx
-          osx_image: xcode8.3
+          osx_image: xcode8.3  # This is the latest working xcode image with osxfuse compatibility; later images come with an OS X version which doesn't allow kernel extensions
           env: TOXENV=py35
         - language: generic
           os: osx
           osx_image: xcode8.3
           env: TOXENV=py37
+        - language: generic
+          os: osx
+          osx_image: xcode9.4
+          env: TOXENV=py35 SKIPFUSE=true
+        - language: generic
+          os: osx
+          osx_image: xcode9.4
+          env: TOXENV=py37 SKIPFUSE=true
+        - language: generic
+          os: osx
+          osx_image: xcode10.3
+          env: TOXENV=py35 SKIPFUSE=true
+        - language: generic
+          os: osx
+          osx_image: xcode10.3
+          env: TOXENV=py37 SKIPFUSE=true
+        - language: generic
+          os: osx
+          osx_image: xcode11.3
+          env: TOXENV=py35 SKIPFUSE=true
+        - language: generic
+          os: osx
+          osx_image: xcode11.3
+          env: TOXENV=py37 SKIPFUSE=true
```

Comment:
and we take this to test "new world".

we only have 2 parallel macOS builds, so guess we ONLY take the old world and new world test runs and remove all the intermediate ones again.

Reply:
I just added them because with `fast_finish: true` it doesn't really matter if the `os: osx` tests take 3 hours. So you'd like to keep only `xcode8.3` and the most recent one?

Reply:
Yes, because I don't want to put too much load onto the macOS infrastructure - it is slow as hell already anyway. Also I don't want to wait ages until it shows results.

Reply:
Fair enough. Will change it.